openslides/utils/models.py

```diff
@@ -39,3 +39,18 @@ def get_root_rest_url(self):
         root_instance = self.get_root_rest_element()
         rest_url = '%s-detail' % type(root_instance)._meta.object_name.lower()
         return reverse(rest_url, args=[str(root_instance.pk)])
+
+    @classmethod
+    def get_collection_name(cls):
+        """
+        Returns the name of the collection.
+
+        This is usualy the name of the django app and the name of the model
+        combined with a slash.
+
+        The collection_name is used to identify an object inside of
+        openslides. For example in the rest api or in the search.
+        """
+        return "{app_name}/{model_name}".format(
+            app_name=cls._meta.app_label,
+            model_name=cls._meta.model_name)
```

Comment:
In line 40 we use `_meta.object_name.lower()` which should be the same as here I think.